rviz_common/CMakeLists.txt

```diff
@@ -57,8 +57,8 @@ configure_file(src/rviz_common/env_config.hpp ${ENV_CONFIG_HPP} @ONLY)
 
 # TODO(jsquare): Adopt parameters given here
 set(ENV_CONFIG_CPP ${CMAKE_CURRENT_BINARY_DIR}/src/rviz_common/env_config.cpp)
-set(RVIZ_VERSION "Alpha")
-set(ROS_DISTRO "ROS 2.0")
+set(RVIZ_VERSION "8.6.0")
```

Comment:
https://github.com/ament/ament_cmake/blob/84e1cea2df62102509de530a5b1bb6d8cde2b5a2/ament_cmake_core/cmake/core/ament_package_xml.cmake#L29

CMake utility to get package.xml version

Reply:
Fantastic, thanks.  I've incorporated it into 0990d59